src/inverseRobot.js

```diff
@@ -8,6 +8,18 @@
 
 function inverseRobot(robot) {
   // write code here
+  const inversedRobot = {};
+
+  for (const key in robot) {
+    const inversedKey = robot[key];
```

Comment:
Try to make up a better name for this variable. Don't use the word 'key' in the name